action.yml

```diff
@@ -1,5 +1,5 @@
 name: 'Create secret in Kubernetes cluster'
-description: 'Create a generic secret or docker-registry secret in Kubernetes cluster'
+description: 'Create a generic secret or docker-registry secret in a Kubernetes such as Azure Kubernetes Service (AKS) clusters'
```

Comment:
Including?

Reply:
Had to change it as the description apparently has a 140 char limit. And adding "including but not limited to" was exceeding it